packages/ContentAutotagging/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@memberjunction/content-autotagging",
+  "version": "1.0.0",
+  "description": "MemberJunction Content Autotagging Application",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "workspaces": [
+    "Core", 
+    "Engine", 
+    "LocalFileSystem",
+    "RSSFeed",
+    "Websites"
+  ],
+  "scripts": {
+    "start": "ts-node-dev src/index.ts",
```

Comment:
`ts-node-dev` watches for changes, so it can be handy for actively developing. For actually running a module, `ts-node` is more appropriate